### PR TITLE
Update WinUI 2 versioning support for WinUI 2.8

### PR DESCRIPTION
**Pull request by gabbybilka**
Linked issue: https://github.com/microsoft/microsoft-ui-xaml/issues/7525

### CODE REVIEW

**Comment by Jak-MS** [comment]
@GrantMeStrength 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by gabbybilka** [comment]
@Karl-Bridge-Microsoft FYI

File: hub/apps/winui/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following table highlights some of most significant differences between WinU
...
 | WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control | WinUI 2 supports the [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and starting with WinUI 2.7 prerelease, the [WebView2](/microsoft-edge/webview2/) control on Desktop. The WebView2 control will come to a WinUI 2.x stable release in the future. |
-| WinUI 3 works downlevel to Windows 10 October 2018 Update (Version 1809, OS build 17763). | WinUI 2 works downlevel to Windows 10 Creators Update (Version 1703, OS build 15063). |
+| WinUI 3 works downlevel to Windows 10 October 2018 Update (Version 1809, OS build 17763). | WinUI 2.0 - 2.7 works downlevel to Windows 10 Creators Update (Version 1703, OS build 15063). WinUI 2.8+ works downlevel to Windows 10 October 2018 Update (Version 1809, OS build 17763). |
```
Comment (Karl-Bridge-Microsoft) [inline hub/apps/winui/index.md]: "WinUI 2.8 and later works downlevel to..."

**Comment by PRMerger7** [comment]
@gabbybilka : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
@gabbybilka - Ready to merge?

**Comment by gabbybilka** [comment]
@Karl-Bridge-Microsoft Ready to merge!

**Pull request merged**